docs/user/installation.rst

```diff
@@ -121,6 +133,53 @@ PyPi
 
 `intelmqsetup` will create all necessary directories, provides a default configuration for new setups. See the :ref:`configuration` for more information on them and how to influence them.
 
+Docker **with** docker-compose (recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq && cd ~/intelmq``
+
+.. code-block:: bash
+
+   git clone https://github.com/certat/intelmq-docker.git
+
+   sudo docker pull certat/intelmq-full:1.0
```

Comment:
doesn't docker compose automatically pull the image when we do `docker-compose up`?

Reply:
With docker pull we force to pull the latest image from dockerhub.

If you only wanna work with docker-compose, I would recommend using `docker-compose pull`.

[Docker-compose up not pulling](https://github.com/docker/compose/issues/3574)